site/_posts/2020-07-20-breached-password-detection.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: blog-post
+title: Help your users avoid using compromised passwords
+description: How can you easily prevent your users from using passwords that have been cracked and made available for sale on the web?
```

Comment:
"How can you easily prevent your users from using passwords that have been cracked and made available for sale on the web? By enabling breached password detection with FusionAuth."